lockup.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["spec/**/*"]
 
-  s.add_dependency "rails", ">= 3", "< 7.1"
+  s.add_dependency "rails", ">= 3", "< 7.2"
```

Comment:
Should we relax this even further? It's unlikely that any 7.X releases of Rails will break this gem. It would save the maintainer some work.

Reply:
@tijmenb I’m open to that – set it at `< 8.0` to force some testing when the next major version comes along?